Source/WebCore/Modules/gamepad/GamepadManager.cpp

```diff
@@ -131,7 +131,7 @@ void GamepadManager::platformGamepadInputActivity(EventMakesGamepadsVisible even
 
 void GamepadManager::makeGamepadVisible(PlatformGamepad& platformGamepad, HashSet<NavigatorGamepad*>& navigatorSet, HashSet<DOMWindow*>& domWindowSet)
 {
-    LOG(Gamepad, "GamepadManager::makeGamepadVisible - New gamepad '%s' is visible", platformGamepad.id().utf8().data());
+    LOG(Gamepad, "(%u) GamepadManager::makeGamepadVisible - New gamepad '%s' is visible", getpid(), platformGamepad.id().utf8().data());
```

Comment:
Since `getpid` returns `pid_t` and not `int` or `unsigned` this needs a cast to `unsigned` or some other technique.